native/build.gradle

```diff
@@ -16,4 +13,5 @@ repositories {
 dependencies {
     testCompile group: 'junit', name: 'junit', version: '4.12'
     compile 'com.github.jansorg:BashSupport:idea-171.x-SNAPSHOT'
```

Comment:
You are right!, sorry I didn't fully understand Maven versioning.

Will send a PR about this soon.